app/assets/stylesheets/layouts/river.scss

```diff
@@ -17,12 +19,12 @@ body { background-color: #E0E0E0; }
 #header, #nav-bottom { background-color: #ADB5B9; }
 .post-expander { background-color: #A4B0BD; }
 .pagination .current { background: #718EB9; }
-.post-icon, #current-icon-dropdown, .green { background-color: #A9B4BB; }
+.post-icon, #current-icon-dropdown, #reply-icon-selector, .replace-icon, .icons-box, .icon-text { background-color: #A9B4BB; }
 a { color: #3F5970; }
 a:visited { color: #284150; }
 .flash.breadcrumbs, .form-table-ender { background-color: #A3A8A9; }
 .subber { background-color: #B9B6AD; }
-.link-box.green { background-color: #3F5370; }
+.link-box.action-new { background-color: #3F5370; }
```

Comment:
Can we move this up to be with the other `link-box`es? (sorry that it wasn't done before)

Reply:
that is a thing I probably would have done when I stripped river.scss but i can do it now, sure